test/libyul/ewasmTranslationTests/extcodehash.yul

```diff
@@ -7,5 +7,5 @@
 // Trace:
 //   INVALID()
 // Memory dump:
-//      0: 0000000000000000000000000000000011111111000000000000000000000000
+//      0: 0000000000000000000000000000000000000000111111110000000000000000
```

Comment:
Probably this should be also ordered differently in memory, as this refers to `address()` above, ie. `0x00...0011111111`.

Reply:
Hmm.. yeah I was also expecting that these values need to be "right-aligned", but it it was also not the case before. Maybe I don't understand the translation tests very well. What should be the correct value here and why?

Reply:
Not sure why this is happening given in `address.yul` it is correct, but probably some offset pointer is off by 8.